tools/generate.cmd

```diff
@@ -56,7 +56,7 @@ if "%7" == "" (call npm i -g autorest)
 
 :: code generation
 @echo on
-call autorest %configFile% --csharp --csharp-sdks-folder=%sdksFolder% --version=%version% --reflect-api-versions
+call autorest %configFile% --csharp --csharp-sdks-folder=%sdksFolder% --version=%version% --reflect-api-versions --tag=package-2017-10
```

Comment:
@prakharsharma10 you cannot change this file, this will affect the entire repo.
Make this change locally to Cdn specific package file.

Reply:
@shahabhijeet  Yes, I didn't intend to check it in. Have updated it.